diff -urNad thunderbird-orig/thunderbird thunderbird/thunderbird
--- thunderbird-orig/thunderbird	2010-02-17 00:57:34.000000000 +0200
+++ thunderbird/thunderbird	2010-02-28 10:36:25.000000000 +0200
@@ -50,7 +50,7 @@
 #uncomment for debugging
 #set -x
 
-moz_libdir=/usr/local/lib/thunderbird-3.0.2
+moz_libdir=/usr/lib/thunderbird-3.0.2
 
 # Use run-mozilla.sh in the current dir if it exists
 # If not, then start resolving symlinks until we find run-mozilla.sh
