--- ./thunderbird.orig	2006-12-07 09:58:51.000000000 +0200
+++ ./thunderbird	2006-12-20 22:14:37.000000000 +0200
@@ -92,8 +92,8 @@
 #uncomment for debugging
 #set -x
 
-moz_libdir=/usr/local/lib/thunderbird-1.5.0.9
-MRE_HOME=/usr/local/lib/mre/mre-1.5.0.9
+moz_libdir=/usr/lib/thunderbird-VERSION
+MRE_HOME=/usr/lib/mre/mre-VERSION
 
 # Use run-mozilla.sh in the current dir if it exists
 # If not, then start resolving symlinks until we find run-mozilla.sh
