diff -urNad thunderbird-orig/thunderbird thunderbird/thunderbird
--- thunderbird-orig/thunderbird	2012-04-28 23:43:50.000000000 +0300
+++ thunderbird/thunderbird	2012-05-13 12:48:21.000000000 +0300
@@ -50,7 +50,7 @@
 #uncomment for debugging
 #set -x
 
-moz_libdir=/usr/local/lib/thunderbird-12.0.1
+moz_libdir=/usr/lib/thunderbird-12.0.1
 
 # Use run-mozilla.sh in the current dir if it exists
 # If not, then start resolving symlinks until we find run-mozilla.sh
